script.js

```diff
@@ -0,0 +1,9 @@
+function showMenu() {
+  document.querySelector('.nav-desk').classList.toggle('active');
+  document.querySelector('.nav-mobil-img').classList.toggle('active');
+  document.querySelector('.nav-mobil-img-2').classList.toggle('active');
+  document.querySelector('.body').classList.toggle('active');
+  document.querySelector('.menu').classList.toggle('hide');
+  document.querySelector('.close').classList.toggle('show');
+}
+document.querySelector('.nav-desk').addEventListener('click', showMenu());
```

Comment:
- [ ] Kindly make changes from [here](https://www.loom.com/share/dd0b3a7d3e554da9b875e0f7d229ffc8) in the loom video.
- [ ] Also make sure when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed